### DISCUSSION TITLE
Consulta Presentación Clase 6, jueves a la mañana

### DISCUSSION THREAD

**Discussion by Schmidt-mat** [dds-jv/foro#376] [Consultas generales] [answered]
Buenas, estaba buscando la clase 6 (link: https://dds-jv.github.io/bitacoras/2022/jueves-m/clase-06/) para guardarme la ppt que se usó y cuando ingreso al link de presentación me abre un github.dev que trata de abrir un pdf pero lo abre mal, está bien el link?

**Comment by gastonprieto** [comment, marked as answer]
hola @Schmidt-mat 

listo, ya esta arreglado el link, gracias por el aviso
